src/box/tuple.h

```diff
@@ -313,6 +313,11 @@ enum tuple_flag {
 	 * be clarified by transaction engine.
 	 */
 	TUPLE_IS_DIRTY = 1,
+	/**
+	 * The tuple belongs to a temporary so it can be freed immediately
```

Comment:
> belongs to a temporary
To a temporary what? I guess you miss 'format' word..

Reply:
Oops. Should be temporary space. Fixed. Thanks for catching this!